telethon/network/mtprotosender.py

```diff
@@ -386,20 +386,27 @@ def _start_reconnect(self):
             data = self._state.encrypt_message_data(data)
             try:
                 await self._connection.send(data)
-            except ConnectionError:
-                self._log.info('Connection closed while sending data')
-                self._start_reconnect()
-                return
+            except ConnectionError as e:
+                __log__.error('Connection closed while sending data')
+                # TODO we will reconnect in _recv loop
+                # this will cause double reconnection
+                # self._start_reconnect()
+                error = e
```

Comment:
`error` seems to be defined only if an exception occurred. In the opposite case this will lead to `NameError` on attempt to use this value.